Example/R87AttributedString/R87AttributedString.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  R87AttributedString.swift
+//  R87AttributedString
+//
+//  Created by Bruno Muniz Azevedo Filho on 27/10/17.
+//  Copyright © 2017 bmaf. All rights reserved.
+//
+import UIKit
+public extension NSMutableAttributedString {
```

Comment:
This file should not exist in the project. The `R87AttributedString.swift` file should be loaded with CocoaPods and should be removed from the example project's folder completely.